content/en/monitors/monitor_types/watchdog.md

```diff
@@ -13,9 +13,7 @@ further_reading:
 
 ## Overview
 
-[Watchdog][1] is an algorithmic feature for APM that automatically detects application and infrastructure issues, by continuously observing trends and patterns in application metrics looking for atypical behavior. 
-
-**Note**: Watchdog is an APM feature, and Watchdog monitors are only available to APM customers.
+[Watchdog][1] is an algorithmic feature for APM and infrastructure integration metrics. It that automatically detects potential application and infrastructure issues by continuously observing trends and patterns in metrics and looking for atypical behavior.
```

Comment:
I would say `infrastructure metrics` instead of `infrastructure integration metrics`, since Watchdog Infra, does not only rely on integration metrics.